app/javascript/dashboard-syncing-icon.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="loading-icon" v-if="loading"></div>
+</template>
+
+<script>
+/* eslint-disable no-alert */
+import axios from 'axios';
+
+export default {
+  props: ['id'],
+  data() {
+    return {
+      loading: false,
+    };
+  },
+  methods: {
+    checkSync() {
+      const TIMEOUT = 5000;
+      axios.get(`/api/organizations/${this.id}/check_sync`)
+        .then((res) => {
+          console.log(this.loading);
+          if (res.data.state === 'completed') {
+            if (this.loading === true) {
+              location.reload(true);
+            }
+            this.loading = false;
+          } else if (res.data.state === 'executing') {
+            this.loading = true;
+            setTimeout(
```

Comment:
no sería posible hacer `setTimeout(this.checkSync, TIMEOUT)`?

Reply:
quizás tenga que ser algo como `this.checkSync.bind(this)`

Reply:
Ahí quedó mejor!